src/core/client.ts

```diff
@@ -204,7 +205,35 @@ export default class Client {
         notice.backtrace[index].source = source
       })
 
-      this.__send(notice)
+      const payload = this.__buildPayload(notice)
+      this.__transport
+          .send({
+            headers: {
+              'X-API-Key': this.config.apiKey,
+              'Content-Type': 'application/json',
+              'Accept': 'text/json, application/json'
+            },
+            method: 'POST',
+            endpoint: endpoint(this.config.endpoint, '/v1/notices/js'),
+            maxObjectDepth: this.config.maxObjectDepth,
+            logger: this.logger,
+          }, payload)
+          .then(res => {
+            if (res.statusCode !== 201) {
+              runAfterNotifyHandlers(notice, this.__afterNotifyHandlers, new Error(`Bad HTTP response: ${res.statusCode}`))
+              this.logger.warn(`Error report failed: unknown response from server. code=${res.statusCode}`)
+              return
+            }
+            const uuid = JSON.parse(res.body).id
+            runAfterNotifyHandlers(merge(notice, {
+              id: uuid
+            }), this.__afterNotifyHandlers)
+            this.logger.info(`Error report sent ⚡ https://app.honeybadger.io/notice/${uuid}`)
+          })
+          .catch(err => {
+            this.logger.error('Error report failed: an unknown error occurred.', `message=${err.message}`)
+            runAfterNotifyHandlers(notice, this.__afterNotifyHandlers, err)
+          })
```

Comment:
All this was duplicated because we had to replicate same behavior in both server and browser.